tasks/main.yml

```diff
@@ -7,6 +10,10 @@
        dest={{ otp_bin_dir }}
        version={{ otp_version }}
 
+- name: Set OpenTripPlanner Session Timeout
+  lineinfile: dest={{ otp_bin_dir }}/src/client/WEB-INF/web.xml regexp="(^|\W)<session-timeout>30</session-timeout>$" line="\t\t<session-timeout>{{ otp_session_timeout }}</session-timeout>"
```

Comment:
Does it make sense to template `web.xml`? Mainly asking because:

1. There may be other things in there we'd want to expose as configuration parameters
2. `lineinfile` has burned me before with regard to changes like this